app/services/service_provider_updater.rb

```diff
@@ -1,9 +1,10 @@
 class ServiceProviderUpdater
   PROTECTED_ATTRIBUTES = [
-    :id,
+    :approved,
```

Comment:
@zachmargolis is correct -- the `approved` flag should be a pass-through from the dashboard.